cmd/awscollector/Dockerfile

```
@@ -1,19 +1,59 @@
+# The binary source type (options: build, copy)
+ARG BIN_SRC_TYPE=build
+
 ################################
-#	Building Stage         #	
-#			       #	
+#	Certificate Stage      #
+#			       #
 ################################
-FROM alpine:latest as build 
+FROM alpine:latest AS certs
 
-# update cert
 RUN apk --update add ca-certificates
 
+################################
+#	Building Stage         #
+#			       #
+################################
+FROM golang:1.17 AS prep-build
+
+ENV GOPROXY=direct
+
+ARG TARGETARCH
+
+# download go modules ahead to speed up the building
+WORKDIR /workspace
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+
+# copy source
+COPY . .
+
+# build
+RUN make ${TARGETARCH}-build-only
```

```suggestion
RUN make ${TARGETARCH}-build
```

Not sure we should encourage skipping the lint checks.  If that's really necessary then a binary can be built externally and copied in.

---

Makes sense. Changed it back to `${TARGETARCH}-build`.